tests/integration/Core/Repository/NotificationServiceTest.php

```diff
@@ -77,6 +77,23 @@ public function testMarkNotificationAsRead()
         $this->assertFalse($notification->isPending);
     }
 
+    /**
+     * @covers \Ibexa\Contracts\Core\Repository\NotificationService::markNotificationAsUnread()
+     */
+    public function testMarkNotificationAsUnread(): void
+    {
+        $repository = $this->getRepository();
+
+        $notificationId = $this->generateId('notification', 5);
+        $notificationService = $repository->getNotificationService();
+
+        $notification = $notificationService->getNotification($notificationId);
```

Comment:
I would make sure that the notification is not marked as unread before running the new method to see the actual effect:
```suggestion
        $notification = $notificationService->getNotification($notificationId);
        
        self::assertFalse($notification->isPending);
```